GitUI/CommandsDialogs/SettingsDialog/CommonLogic.cs

```diff
@@ -54,6 +54,22 @@ public CommonLogic(GitModule module)
         public const string GitExtensionsShellEx32Name = "GitExtensionsShellEx32.dll";
         public const string GitExtensionsShellEx64Name = "GitExtensionsShellEx64.dll";
 
+        public string GetGlobalDiffTool()
```

Comment:
Why duplicate the handling of diff.guitool?
Already in GetDiffToolFromConfig()

Reply:
I tried to explain why in the commit message - "extending CommonLogic so diff and merge is treated similarly" - but I can elaborate: I'm a big fan of consistency. Since there is already `GetGlobalMergeTool`, `SetGlobalMergeTool` and `IsMergeTool`, I don't see why there shouldn't be similar functions for Diff. Another option would be to remove these Merge functions - that is also consistent. I think these functions are nice to have.

Reply:
Probably this (similar for merge.tool) that should return
CheckSettingsLogic.GetDiffToolFromConfig(CheckSettingsLogic.CommonLogic.ConfigFileSettingsSet.GlobalSettings)

Reply:
Is it the implementation of  GetGlobalDiffTool (and hence  GetGlobalMergeTool) that is the problem or the whole thing? Anyway, I can update the PR and remove these changes since it's not really related to the issue. I just thought it was nice to clean up things ;-)